javascript/node/selenium-webdriver/chrome.js

```diff
@@ -150,19 +151,23 @@ class ServiceBuilder extends chromium.ServiceBuilder {
   /**
    * @param {string=} opt_exe Path to the server executable to use. If omitted,
    *     the builder will attempt to locate the chromedriver on the current
-   *     PATH.
+   *     PATH. If the chromedriver is not available in path, selenium-manager will
+   *     download the chromedriver
    * @throws {Error} If provided executable does not exist, or the chromedriver
    *     cannot be found on the PATH.
    */
   constructor(opt_exe) {
     let exe = opt_exe || locateSynchronously()
+
     if (!exe) {
-      throw Error(
-        `The ChromeDriver could not be found on the current PATH. Please ` +
-          `download the latest version of the ChromeDriver from ` +
-          `http://chromedriver.storage.googleapis.com/index.html and ensure ` +
-          `it can be found on your PATH.`
+      console.log(
+        ` The ChromeDriver could not be found on the current PATH, trying Selenium Manager`
       )
+      exe = driverLocation(Browser.CHROME)
+    }
+
+    if (!exe) {
+      throw Error(`Unable to obtain driver using Selenium Manager`)
```

Comment:
There's nothing a user can do to fix a Selenium Manager problem, but they can fix their driver problem by adding chromdriver to PATH.. We want to log Selenium Manager problems in Selenium manager code, but the driver classes should throw the same error they've always thrown.